utility/utils.py

```diff
@@ -2798,3 +2800,31 @@ def is_unsecured_registry(test_data):
             return insecure_registry_value  # Return boolean value for insecure-registry
 
     return False  # Default return value if conditions are not met
+
+
+def restart_rgw_and_wait(rgw_node, rgw_service_name):
+    """
+    Restart the given RGW service and wait until all daemons are running.
+    """
+    log.info(f"Restarting RGW service: {rgw_service_name}")
+    rgw_node.exec_command(sudo=True, cmd=f"ceph orch restart {rgw_service_name}")
+
```

Comment:
As a future improvement, you may want to add a sleep here for half second... so that we allow the daemon to gracefully be restarted.